### ISSUE TITLE
background image should be cached until a login process has been finshed

### ISSUE THREAD

**Issue by j-ed** [nextcloud/unsplash#2]
After activating that app I realized that a grey screen is shown for a short moment before the downloaded picture is displayed. I think it might be better to show the previous picture before replacing it by a new one, to prevent grey screens.
Additionally the picture changes *really* on every page refresh/reload. This is a little bit confusing/disturbing e.g. when you have the Two Factor U2F app activated and a different background picture is display right after you've entered your credentials and waiting for the confirmation of the U2F token. It would be nice if a background picture is cached at least as long as it takes to finish o single login process.

**Comment by jancborchardt** [comment]
So there's two points here:
- About the grey screen I disagree. It would be way more jarring to display the previous image for a second and then switch to the new one. The current grey is very neutral for that purpose.

- And about it really switching every time: Well, it is supposed to be a showcase of pictures. :) And when do you switch, when do you not? Time limit, interactions, clicks? There are also other interstitials like when you click links in the Mail app or update Nextcloud.

**Comment by jancborchardt** [comment]
This is related to Request: Pull image(s) in advance with cronjob to have it ready uppon loading the page #7

**Comment by MarkWieczorek** [comment]
I really like the concept of this app. However, I disabled it after a few hours of usage because I found it a bit annoying that the image changed every time you opened a new page, or switched from one app to another. For people like me (perhaps I am a minority...), it would be great to have an option like "Use a single image per login session", or "Reload a new image every X minutes". I agree with the statement above the using a different image for the login and 2FA is perhaps the worst offender in this regard. Otherwise, its great!
